docs/Getting Started/Debian/Debian Bullseye Root on ZFS.rst

```diff
@@ -292,7 +292,7 @@ Step 2: Disk Formatting
 
    - Unencrypted::
 
-       zpool create \
+       zpool create -f \
```

Comment:
See above: You don't want garbaged filesystem, just make clear, that you don't use this partition